WordPress/gutenberg
# Site Logo: add padding and margin support

Related:

- https://github.com/WordPress/gutenberg/issues/43241
- https://github.com/WordPress/gutenberg/issues/43243

## What?

Add padding and margin support to the Site Logo block.

## Why?

To create consistency across blocks.

## How?

Added the relevant block support in `block.json` and updated Core Block Reference docs.

## Testing Instructions

1. Go to Appearance > Editor (beta)
2. Add the Site Logo block to the template
3. Configure margin and padding and verify the output.

![site-logo-supports](https://user-images.githubusercontent.com/405912/186159454-b9b62636-0b7c-4389-9f90-06cb57467af9.gif)
